bridge/client/app/_services/data.service.ts

```diff
@@ -70,7 +70,8 @@ export class DataService {
       availableVersions: this.apiService.getAvailableVersions(),
       bridgeInfo: this.apiService.getKeptnInfo(),
       keptnVersion: this.apiService.getKeptnVersion(),
```

Comment:
`apiService.getKeptnVersion()` currently parses the Keptn version from `/swagger-ui/swagger.yaml`, but I see that the `/metadata` endpoint now also provides us that information. Maybe we can get rid of this call. Since #2760 addresses the problem, that we have to many API calls in the bridge, I would post this as a comment on that issue and approve this one.